simvp/datasets/dataloader_cracks.py

```diff
@@ -0,0 +1,91 @@
+from PIL import Image
+import torch
+from torch.utils.data import Dataset
+from .video_dataset import VideoFolder, video_loader, make_dataset, find_videos
+import torchvision.transforms as transforms
+
+
+def load_data(batch_size, val_batch_size, data_root, num_workers=4, pre_seq_length=5, aft_seq_length=1):
+    img_width = 128
+    validation_batch_size = 1
+    num_workers = 4
+    train_set_path = './data/cracks/train'
```

Comment:
Should this be something that uses the `data_root`, like `train_set_path = os.path.join(data_root, "cracks/train")`? (and same for the test_set_path)?